Apps/WebClient/src/Server/Startup.cs

```diff
@@ -1,25 +1,49 @@
+//-------------------------------------------------------------------------
+// Copyright � 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
 #pragma warning disable CA1303 //disable literal strings check
 namespace HealthGateway.WebClient
 {
     using System;
     using System.IdentityModel.Tokens.Jwt;
     using System.IO;
+    using System.Reflection;
     using System.Security.Claims;
     using System.Threading.Tasks;
     using HealthGateway.WebClient.Services;
+    using HealthGateway.WebClient.Swagger;
     using Microsoft.AspNetCore.Authentication.Cookies;
     using Microsoft.AspNetCore.Authentication.OpenIdConnect;
     using Microsoft.AspNetCore.Builder;
     using Microsoft.AspNetCore.Hosting;
     using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.HttpOverrides;
+    using Microsoft.AspNetCore.Mvc;
     using Microsoft.AspNetCore.ResponseCompression;
     using Microsoft.AspNetCore.SpaServices.Webpack;
     using Microsoft.AspNetCore.StaticFiles;
     using Microsoft.Extensions.Configuration;
     using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+    using Swashbuckle.AspNetCore.SwaggerUI;
 
+    /// <summary>
+    /// Configures the application during startup.
+    /// </summary>
     public class Startup

```

Comment:
SonarQ its flagging some lines on this method:
https://sonarqube-gateway.pathfinder.gov.bc.ca/project/issues?id=webclient_dev&open=AWwmE1CeZJzGytDdbsgL&resolved=false&severities=CRITICAL&types=CODE_SMELL

Are those addressable now?

Reply:
I don't think we can refactor this yet but once we look at doing auth in Vue most of it should go away.